package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo-drivex",
-  "version": "0.3.0",
+  "version": "0.4.4",
```

Comment:
sure, was just wondering the choice of patch version, instead of `0.4.4` vs `0.4.2`. Anyways I'm merging this now since its been pending for a while.